modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -1,114 +1,83 @@
-<3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.
-<4> The scrape target for the metrics from port `8443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
-<5> The scrape target for the metrics from port `9443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
-====
-
-. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
-+
 [source,terminal,subs="+attributes,quotes"]
 ----
-$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+oc label namespace {prod-namespace} openshift.io/cluster-monitoring=true
 ----
+====
 
 .Verification
 
-. Use port forwarding to access the `Prometheus` Service locally:
-+
-[source,terminal,subs="+attributes,quotes"]
-----
-$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
-----
-. Verify that all targets are up by viewing the targets endpoint at `localhost:9090/targets`.
-. Use the Prometheus console to view and query metrics:
-** View metrics at `localhost:9090/metrics`.
-** Query metrics from `localhost:9090/graph`.
+. For a fresh installation of {prod-short}, generate metrics by creating a {prod-short} workspace from the Dashboard.
+
+. In the *Administrator* view of the OpenShift web console, navigate to *Observe* -> *Metrics* from the sidebar menu on the left.
+
+. Run a PromQL query to confirm that the metrics are available. For example, enter `devworkspace_started_total` and click *Run queries*.
 +
-For more information, see link:https://prometheus.io/docs/introduction/first_steps/#using-the-expression-browser[Using the expression browser].
+For more metrics, see xref:ref_devworkspace-specific-metrics[].
 
-[role="_additional-resources"]
-.Additional resources
+[TIP]
+====
 
-* link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/[Configuring Prometheus]
+If the metrics are not available, view the Prometheus container logs for possible RBAC-related errors.
 
-* link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Querying Prometheus]
+Get the Prometheus pod name:
+```
+$ oc get pods -l app.kubernetes.io/name=prometheus -n openshift-monitoring -o=jsonpath='{.items[*].metadata.name}'
+prometheus-k8s-0
+```
+According to the output above, there is a Prometheus pod named `prometheus-k8s-0`.
 
-* link:https://prometheus.io/docs/concepts/metric_types/[Prometheus metric types]
+To print the last 20 lines of the Prometheus container logs from the `prometheus-k8s-0` pod:
+```
+$ oc logs --tail=20 prometheus-k8s-0 -c prometheus -n openshift-monitoring
+```
+
+====
```

Comment:
Looks great, I removed the link in line 87 since it is not very relevant for this doc anymore, and I've made a new commit for this suggestion since I was not able to apply it in the GitHub UI